### DISCUSSION TITLE
Vote for next Eclipse JNoSQL release version.

### DISCUSSION THREAD

**Discussion by otaviojava** [eclipse-jnosql/jnosql#373] [General]
Hold a vote to decide whether to release the final version of Eclipse JNoSQL, version 1.0.0. We have been working on beta versions for about two years, and it's about time we release a new final version. We don't need to release a final version with the specification or submit it to the Jakarta EE executive committee.

This project has two Jakarta Specifications as dependencies, Jakarta Data and NoSQL, and it implements them. As with any decision, we need to consider the trade-offs. On the one hand, releasing a final version would allow us to follow the entire process, and we could release a new version with any changes to these dependencies. Even if there are compatibility issues, we can document and update them as Jakarta EE did. On the other hand, there are cons to releasing a final version based on beta specification dependencies. Ultimately, it's up to us to decide whether the pros outweigh the cons.

## Pros of the release
* We can release a final release and follow the entire process
* We can release a new version with any change in these dependencies.
* Even with break compatibility, we can document and update as Jakarta EE did.
* Done is better than perfect. 

## Cons of the release

* Those are based on beta specification dependencies.

Please, define your vote:

1. 1.0.0 final release
2.  Use several beta versions until Jakarta NoSQL and Data became final

![7jp8l8](https://user-images.githubusercontent.com/863011/234894547-55256a8b-01e1-4600-81fb-247ac42d7ddd.jpg)

**Comment by dearrudam** [comment]
I vote for the first option: "1.0.0 final release".

I agree with "done is better than perfect" and, once the beta dependencies changes, we'll need to update the project anyway, am I right? Then, with a "1.0.0" version, I think we will gonna be able to promote the project or at least create awareness around the project, with more consistency, with the interested ones, like communities, companies, and so on...

**Reply by scaramuzzini** [reply]
it makes sense

**Comment by mpredli01** [comment]
I'm a +1 for option 1.  I agree that we should get this released as a 1.0.0 version sooner rather than later. We can always provide milestone updates along the way as Jakarta Data and Jakarta NoSQL evolve this year.  It's not unusual for a milestone release to be the compatible implementation.

**Comment by otaviojava** [comment]
Ok, we voted for version 1.0.0.

So, I'm closing this issue.
Thank you, everyone.
